src/components/views/UserPage.js

```diff
@@ -159,19 +123,19 @@ const UserPage = () =>{
         </div>
     )
 
-    if(preferences){
+    if(user){
         userPreferences = (
             <div>
                 <div className="userPage title">Preferences</div>
                 <table className="userPage table">
                     <tbody>
                     <tr className="userPage player-info-container">
                         <td> Age Preference:</td>
-                        <td className="userPage td"> {preferences.minAge} Years - {preferences.maxAge} Years</td>
+                        <td className="userPage td"> {user.minAge} Years - {user.maxAge} Years</td>
                     </tr>
                     <tr className="userPage player-info-container">
                         <td> Gender Preferences: </td>
-                        <td className="userPage td"> {preferences.genderPreferences.join(', ')} </td>
+                        <td className="userPage td"> {user.genderPreferences.join(', ')} </td>
```

Comment:
the capital letter sort of hurt my eyes 👀

Reply:
I agree, problem is that JS doesn't have a capitalize method. But CSS has it, will see what i can do with it

Reply:
I have applied a temporary fix, that capitalizes them but for gender preferences it only works for the first word. But still looks better than all caps

Reply:
https://www.w3schools.com/jsref/jsref_toLowerCase.asp

Reply:
yes, putting it to lowercase is no problem, but the to capitalize. But i will try some more things.

Reply:
ahh now it works. I put it to lowercase with JS and then capitalize it with scss. Also changed it in the EditProfile where the current gender is displayed

Reply:
ahh now it works. I put it to lowercase with JS and then capitalize it with scss. Also changed it in the EditProfile where the current gender is displayed